src/commands/cmd_server.cc

```diff
@@ -1222,6 +1222,39 @@ class CommandAnalyze : public Commander {
   std::vector<std::string> command_args_;
 };
 
+class CommandApplyBatch : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    raw_batch_ = args[1];
+    if (args.size() > 2) {
+      if (args.size() > 3) {
+        return {Status::RedisParseErr, errWrongNumOfArguments};
+      }
+      if (!util::EqualICase(args[2], "low_pri")) {
```

Comment:
Not sure if LOWPRI is better? As far as I recall, it seems Redis doesn't prefer using underscores in command options.

Reply:
Indeed, I will change it.